conda_smithy/github.py

```diff
@@ -52,6 +52,33 @@ def add_membership(team, member):
     return (headers, data)
 
 
+def has_in_members(team, member):
+    status, headers, data = team._requester.requestJson(
+        "GET",
+        self.url + "/members/" + member
+    )
+    return status == 204
```

Comment:
Should we be raising if there is an error?

Reply:
Also any reason not to use [this]( http://pygithub.readthedocs.io/en/latest/github_objects/Team.html#github.Team.Team.has_in_members )?

Reply:
...or is this suppose to be cached somehow as well?

Reply:
This function checks that the member is in the team, so no need to raise an error.

PyGithub's API requires a User object instead of a username. This way it reduces a GitHub API call

Reply:
Couldn't there still be rate limiting or some other issue that would block this?

Ok, that makes sense.

Reply:
Right. rate-limiting can raise an error